gitautosync/pull_from_remote.py

```diff
@@ -0,0 +1,76 @@
+import os
+import subprocess
+
+
+def pull_from_remote(repo_name, branch_name, sync_path, account, domain):
```

Comment:
Oh yeah, rn there doesn't seem to be a reason. I'll go ahead and change that. I think before there was functionality that formated the domain, account and repo name differently based on if the instructor was using a git auth token or not.